src/adapters/postgres/fields-adapter.ts

```
@@ -2,304 +1,1773 @@
+          const labels = await this.findDynamicOptions(
+            data.sourceDetails.table,
+            `value='${data.value}'`
+          );
+          if (labels && labels.length > 0) {
+            processedValue = labels[0].name;
+          }
         }
-        return {offset,limit,whereClause};
       }
 
+      delete data.fieldParams;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)